hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractRevisionTest.php

```diff
@@ -1,32 +1,26 @@
 <?php
 
-require_once 'CiviTest/CiviUnitTestCase.php';
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
 
 /**
  * Class CRM_Hrjobcontract_BAO_HRJobContractRevisionTest
+ *
+ * @group headless
  */
-class CRM_Hrjobcontract_BAO_HRJobContractRevisionTest extends CiviUnitTestCase {
+class CRM_Hrjobcontract_BAO_HRJobContractRevisionTest extends PHPUnit_Framework_TestCase implements
+  HeadlessInterface,
+  TransactionalInterface {
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()->installMe(__DIR__)->apply();
+  }
+
   function setUp() {
-    parent::setUp();
-    $this->quickCleanup(array(
-      'civicrm_hrjobcontract_details',
-      'civicrm_hrjobcontract_health',
-      'civicrm_hrjobcontract_role',
-      'civicrm_hrjobcontract_hour',
-      'civicrm_hrjobcontract_pay',
-      'civicrm_hrjobcontract_leave',
-      'civicrm_hrjobcontract_pension',
-      'civicrm_hrjobcontract_revision',
-      'civicrm_hrjobcontract',
-    ));
     $upgrader = CRM_Hrjobcontract_Upgrader::instance();
     $upgrader->install();
```

Comment:
Since this is part of the extension installation, you can move those two lines to the setupHeadless method. It will avoid running the upgrader after every test method and will probably save you a few precious seconds